src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/HtmlToTextUtilsTests.cs

```diff
@@ -215,6 +215,34 @@ public async Task HtmlToText_OrderedList_HasLineAfter()
             Snapshot.Match(text);
         }
 
+        [Fact]
+        public async Task HtmlToText_OrderedList_OverTenItemsWithMultiline()
+        {
+            var text = await HtmlToTextUtils.HtmlToText(
+                @"
+                <ol>
+                    <li>List item 1</li>
+                    <li>List item 2</li>
+                    <li>List item 3</li>
+                    <li>List item 4</li>
+                    <li>List item 5</li>
+                    <li>List item 6</li>
+                    <li>List item 7</li>
+                    <li>List item 8</li>
+                    <li>
+                        <p>List item 9</p>
+                        <p>Over multiple lines</p>
+                    </li>
+                    <li>List item 10</li>
+                    <li>
+                        <p>List item 10</p>
```

Comment:
Just a little typo here - should be 11 :)

Reply:
Good catch!